app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -445,6 +477,8 @@ void setFullscreen(boolean isFullscreen) {
             params.height = ViewGroup.LayoutParams.MATCH_PARENT;
         } else {
             reset();
+            // We'll zoom out until it returns false, which means it has min zoom level
+            while (mWebView.zoomOut()) {}
```

Comment:
I hope this work as expected and we wouldn't end up in infinite loop here 😬

Reply:
According to API docs - it should, but you're right - it's damn dangerous if it doesn't! Maybe add some artificial limit - like do that 30 times, and then force break it.